extension.yaml

```diff
@@ -294,7 +294,7 @@ params:
     validationErrorMessage: Firestore collection paths must be an odd number of segments separated by slashes, e.g. "path/to/firestore_collection".
     default: path/to/firestore_collection
     required: true
-    immutable: true
+    immutable: false
```

Comment:
Could you remove this key completely like here: https://github.com/firebase/extensions/blob/f59076ff5461f4426a960219d7df21a937893ec1/firestore-bigquery-export/extension.yaml#L348C12-L363